groups/sig-release/groups.yaml

```diff
@@ -374,35 +376,38 @@ groups:
       - priyankasaggu11929@gmail.com # 1.29 Lead
       - xandergrzyw@gmail.com # 1.29 Emeritus Advisor
     members:
-      - mehabhalodiya@gmail.com # 1.29 Lead Shadow
+      - abigailkmccarthy@gmail.com # 1.29 Communications Shadow
+      - AnaMMedina21@gmail.com # 1.29 Enhancements Shadow
+      - bradmccoydev@gmail.com # 1.29 Bug Triage Shadow
+      - drewhagendev@gmail.com # 1.29 Docs Shadow
+      - egbunaoluebube@gmail.com # 1.29 Docs Shadow
+      - faeka6@gmail.com # 1.29 Release Notes Shadow
+      - harshitasao@gmail.com # 1.29 Docs Shadow
+      - hkamel.msft@gmail.com # 1.29 Communications Shadow
+      - james@quigley.us # 1.29 Communications Shadow
+      - kcmartin2@mac.com # 1.29 Communications Shadow
       - m.r.boxell@gmail.com # 1.29 Lead Shadow
+      - mankulka@redhat.com # 1.29 CI Signal Shadow
+      - markrossetti@gmail.com # 1.29 Release Manager associate
+      - marosset@microsoft.com # 1.29 Release Manager associate
+      - maryam_tavakkoli@hotmail.com # 1.29 Bug Triage Shadow
+      - mehabhalodiya@gmail.com # 1.29 Lead Shadow
+      - mengjiao.liu@daoloud.io # 1.29 Release Notes Shadow
```

Comment:
```suggestion
      - mengjiao.liu@daocloud.io # 1.29 Release Notes Shadow
```

Reply:
Thanks for pointing. Corrected in the recent commit refresh.